LeGO-LOAM/src/featureAssociation.cpp

```diff
@@ -38,7 +38,9 @@ class FeatureAssociation{
 
 private:
 
-	ros::NodeHandle nh;
+    enum { SIZE_SCANS = N_SCAN * Horizon_SCAN };
+
+    ros::NodeHandle nh;
```

Comment:
Would it make sense to use a reference to the nodehandle instead of creating a new one?

Reply:
Most definitively, but it is not the purpose of this PR.

I think it is even more important to achieve a good library / node separation first.

But again, it is not the purpose of this PR